kubespider/source_provider/youtube_source_provider/provider.py

```diff
@@ -53,7 +53,7 @@ def is_webhook_enable(self) -> bool:
 
     def should_handle(self, event: Event) -> bool:
         parse_url = urlparse(event.source)
-        if parse_url.hostname == 'www.youtube.com':
+        if parse_url.hostname in ('www.youtube.com', 'youtube.com', 'm.youtube.com', 'youtu.be'):
```

Comment:
also need to add the same value of ytdlp_download_provider handle_host default value

Reply:
prefer to make this configurable

Reply:
> also need to add the same value of ytdlp_download_provider handle_host default value

source provider is not bind to downloader porvider , so YouTube source provider better not read the yt-dlp's config

Reply:
> prefer to make this configurable

yt-dlp also have config , in this situation, we need consider one better solution . for now , just write in code